src/mappings/entities/mapPolyxTransaction.ts

```diff
@@ -39,11 +41,38 @@ const handleTreasuryReimbursement = async (args: HandlerArgs): Promise<void> =>
 
   const did = getTextValue(rawIdentity);
   const identity = await Identity.get(did);
+
+  // treasury reimbursement is only 80% of the actual amount deducted
+  const reimbursementBalance = new BigNumber(getTextValue(rawBalance) || 0);
```

Comment:
I don't get this. Why wouldn't it be 100%, and why wouldn't the actual amount not be on the event.

Reply:
it was like this before 5.4.1. Not sure why, but they changed it since 5.4.1